### PR TITLE
deps update (#3800)

### PR DESCRIPTION
**Pull request by jmaslek**
Need to bump requirements for yf.

Building the installers now to confirm

Following commands were run:
```
poetry add yfinance@latest
poetry update
```

### CODE REVIEW

**Comment by jmaslek** [comment]
statsmodels needs to be at 0.13.2 because of scipy

File: pyproject.toml [hunk 1]
```diff
@@ -44,7 +44,7 @@ pycoingecko = "^2.2.0"
 detecta = "^0.0.5"
 tradingview-ta = "^3.2.3"
 finvizfinance = "^0.14.0"
-statsmodels = "^0.13"
+statsmodels = "0.13.2"
```
Comment (hjoaquim) [inline pyproject.toml:47]: @jmaslek can we have `statsmodels = ">=0.13.2"` so we don't pin an exact version or this will cause scipy to break?

Comment (jmaslek) [reply pyproject.toml:47]: Causes the dep tree to break.

**Comment by hjoaquim** [comment]
Fix #3796

**Comment by jmaslek** [comment]
okay i got all the tests to pass (hence the 1Mil net adds). 

however the installers are not building correctly.  and the anaconda builds are struggling.

**Review by piiq — APPROVED** [review]
I've built an M1 installer of this branch and it works.
The windows installer has just finished building and @andrewkenreich will have a look if it passes the smoke test on a windows machine. If so - we're good to merge this one

File: tests/openbb_terminal/etf/txt/test_etf_controller/test_call_candle.txt [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+  Right contains 3 more items, first extra item: ('interval', '1d')
+  Use -v to get the full diff
+
+- ARKW: No data found for this date range, symbol may be delisted
+
+1 Failed download:
+- ARKW: No data found for this date range, symbol may be delisted
+ETF ticker provided does not exist!
+
+No ticker loaded. First use `load {ticker}`
```
[the remark this answers is not in the document]
Comment (andrewkenreich) [reply tests/openbb_terminal/etf/txt/test_etf_controller/test_call_candle.txt:23]: works!

**Review by andrewkenreich — APPROVED** [review]
working for me on windows

**Pull request merged**